rfcs/0062-content-addressed-paths.md

```diff
@@ -60,22 +60,75 @@ cutoffs.
 In all that follows, we pretend that each derivation has only one output.
 This doesn't change the reasoning but makes things easier to state.
 
-The gist of the design is that
+The gist of the design is that:
 
 - Some derivations can be marked as content-adressed (ca), in which case their
   output will be moved to a path `ca` determined only by its content after the
   build
-- Each (non content-adressed) derivation will have two outputs: A `static` one
-  computed at evaluation time and a `dynamic` one computed from the dynamic
-  outputs of its dependencies. These outputs may be identical if the derivation
-  doesn't (transitively) depend on any ca derivation
-- just prior to being realized, each derivation gets rewritten by replacing
-  each of its dependencies by its `dynamic` or `ca` path
+- We introduce the notion of a `resolved derivation` which is a derivation that
+  doesn't refer to any other derivation but only to concrete store paths.
```

Comment:
Unless we commit to breaking self cycles in the first version, we cannot actually do this because there is no hash of normal derivations with cyclic data.

> Derivations that don't transitively depend on any ca derivation are “equivalent” to their associated resolved derivation in that they refer to the same inputs and have the same output hash.

This quote in particular is in conflict with the above.

I think you wish to say that a `resolved` derivation depends on no content-addressed or fixed output derivations, but only concrete store paths and normal derivations.